test/TestProjects/Spread-Cadl/Spread-Cadl.cadl

```diff
@@ -78,25 +78,25 @@ alias ThingAliasWithOptionalProps = {
 @route("/spreadModel")
 @doc("spread a model as body.")
 @post
-@convenientAPI(true)
+@convenienceAPI
```

Comment:
this is the old decorator which we shouldn't use anymore we need to remove support for this